_posts/2022-05-12-jtbd-in-openmic.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title:  "Jobs to be done in Open Mic Session"
+driveId: 1kTtGdCpnt3ROwvLGr4c-8-crPf-uPGcs/preview
+category: "open-mic"
+#excerpt: "Open mic session about Jobs to be Done, includes presentation, video and further reading."
+---
+
+On friday 13th May at 10:30 took place Open mic session. Today\'s topic is \"Jobs to be Done\". Video and presentation included.
```

Comment:
`On Friday 13th May at 10:30 an open Mic session took place. The topic was \"Jobs to be Done\". Video and presentation included.`